microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -271,6 +280,9 @@ def verify_gpu_cuda_with_pytorch(
         pythonvenv_path = work_path + "/gpu_pytorch"
         pythonvenv = node.tools.create(PythonVenv, venv_path=pythonvenv_path)
 
+        if isinstance(node.os, Linux):
```

Comment:
Add comments to explain. If the os disk is not enough, you can use resource disk to save files.

Reply:
The disk full issue happens while the .whl package is being downloading by pip (the tmp files being stored in root disk).

Facing the issue even after specifying cace dir in pip

 envs = {"TMPDIR": str(cache_dir)}
        cmd_result = self.run(
            f"-m pip install -q {packages_name} --cache-dir={cache_dir}",
            force_run=True,
            update_envs=envs,
        )

Reply:
Thank you for the explanation, please add the comment in the code. So we will know why it needs to clean in future.

Reply:
Added comment, raised package cleanup as a separate PR 
https://github.com/microsoft/lisa/pull/3200